app/views/offer/new.html.erb

```diff
@@ -0,0 +1,31 @@
+<table>
+    <tr>
+      <td><%= @product.name %></td>
+      <td><%= @product.description %></td>
+      <td><%= @product.suggested_price %></td>
+      <td><%= @product.merchantprofile_id %></td>
+    </tr>
+</table>
+
+<%= form_with scope: :offer, url: new_offer_path, local: true do |form| %>
```

Comment:
`new_offer_path` to nie jest odpowiedni url formularza, bo prowadzi do akcji `new`, a nie do `create`